### ISSUE TITLE
Update requirements.txt

### ISSUE THREAD

**Issue by jekwatt** [BCM-HGSC/pycon2020_excel_poster#17]
If you are using Jupyter Lab, you will want to also ensure that ipywidgets is installed:
```
# only if you don't have ipywidgets installed
conda install -c conda-forge ipywidgets

# the next line is necessary
jupyter labextension install @jupyter-widgets/jupyterlab-manager
```
Users can install the current version of ipywidgets with pip or conda.
https://ipywidgets.readthedocs.io/en/latest/user_install.html

**Comment by jekwatt** [comment]
- `requirements.txt` is for specifying pip dependencies .
- `pip` keys map to packages that need to be installed via `pip`.
- Instead of listing out all `pip` dependencies, have one line in `environment.yml` file under `pip`:
```
dependencies:
...
- pip:
   - -r file:requirements.txt
```
- Gives nice separation between conda environment and pip install requirements.

**Comment by jekwatt** [comment]
Issue resolved via #21.
